src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/request/UserProfileCreationRequest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.request;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Set;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+@Setter
+@Getter
+@Builder(builderMethodName = "anUserProfileCreationRequest")
+public class UserProfileCreationRequest  {
+
+    private String email;
+
+    @NotBlank
+    private String firstName;
+
+    @NotBlank
+    private String lastName;
+
+    @NotBlank
```

Comment:
no beans needed..as mentioned above comments we had already built those in 1952/1953. As part of your ticket for now we expect yo to use hibernate domain object and this duplication.

Reply:
This is for UP call and i don't think this is same as mentioned in the above ticket.